apps/fluent-tester/src/TestComponents/Drawer/DrawerDefault.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { Avatar } from '@fluentui-react-native/avatar';
+import { ButtonV1 as Button } from '@fluentui-react-native/button';
+import { Drawer } from '@fluentui-react-native/drawer';
+import type { DrawerPositionType } from '@fluentui-react-native/drawer';
+import { Stack } from '@fluentui-react-native/stack';
+import { Switch } from '@fluentui-react-native/switch';
+import { Text } from '@fluentui-react-native/text';
+
+import { stackStyle } from '../Common/styles';
+
+export const DrawerDefault: React.FunctionComponent = () => {
+  const [isDrawerVisible, setIsDrawerVisible] = React.useState(false);
+  const [drawerPosition, setDrawerPosition] = React.useState<DrawerPositionType>('left');
+
+  const handleDrawerClose = React.useCallback(() => {
+    setIsDrawerVisible(false);
+  }, []);
+
+  const handleDrawerOpen = React.useCallback((position) => {
+    setDrawerPosition(position);
+    setIsDrawerVisible(true);
+  }, []);
+
+  const handleChildrenClick = React.useCallback(() => {
+    console.log('Children Clicked');
+  }, []);
+
+  return (
+    <Stack style={stackStyle}>
+      <View style={drawerContentstyles.container}>
+        <View style={drawerContentstyles.content}>
+          <Text>This is the main content of the screen.</Text>
+          <Button appearance="outline" onClick={() => handleDrawerOpen('left')}>
+            Open Drawer (Left)
+          </Button>
+          <Button appearance="outline" onClick={() => handleDrawerOpen('right')}>
+            Open Drawer (Right)
+          </Button>
+          <Button appearance="outline" onClick={() => handleDrawerOpen('bottom')}>
+            Open Drawer (Bottom)
+          </Button>
+        </View>
+        <Drawer visible={isDrawerVisible} onClose={handleDrawerClose} onBackdropClick={handleDrawerClose} position={drawerPosition}>
+          <View style={drawerContentstyles.drawerContent}>
+            <View style={drawerContentstyles.flexRow}>
+              <Avatar active={'active'} activeAppearance="ring" size={56} name="John Doe" avatarColor={'colorful'} />
+              <Text variant="headerSemibold" style={drawerContentstyles.text}>
+                John Doe
```

Comment:
Is there any restriction on what can be placed within drawer from design guidance?

Reply:
Now Developers can put anything inside drawer and they handle every styles & layout how it looks. 
There are guidance for the width,height of Drawer that will be tokens only but users has ability to modify it.
It is treated as a blackbox and we will match what native android has implemented.

Reply:
Discussed this on this over spec review.

Reply:
Please add a note to validate different content sizes an their impact on the drawer size